pepper-apis/studybuilder-cli/studies/basil/tabular-activity.conf

```diff
@@ -0,0 +1,66 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": ${id.act.tabular},
+  "versionTag": "v1",
+  "displayOrder": 3,
+  "nameTemplate": {"templateText": "$tabular.name"},
+  "titleTemplate": {"templateText": "$tabular.title"},
+  "sections": [
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "headers": [
+            {
+              "columnSpan":1,
+              "label": {"templateText": "$tabular.header.column_left"}
+            },
+            {
+              "columnSpan":1,
+              "label": {"templateText": "$tabular.header.column_right"}
+            }
+          ],
+          "rows": [
+            {
+              "questions": [
+                {
+                  include required("../snippets/decimal-question.conf"),
+                  "stableId": "TABULAR_QUESTION_L1",
+                  "promptTemplate": {"templateType": "TEXT", "templateText": "$tabular.question.l1"},
+                  "validations": []
+                },
+                {
+                  include required("../snippets/decimal-question.conf"),
+                  "stableId": "TABULAR_QUESTION_R1",
+                  "promptTemplate": {"templateType": "TEXT", "templateText": "$tabular.question.r1"},
+                  "validations": []
+                }
+              ]
+            },
+            {
+              "questions": [
+                {
+                  include required("../snippets/decimal-question.conf"),
+                  "stableId": "TABULAR_QUESTION_L2",
+                  "promptTemplate": {"templateType": "TEXT", "templateText": "$tabular.question.l2"},
+                  "validations": []
+                },
+                {
```

Comment:
Is it a valid scenario if you have only 1 question in one of the rows rather than 2 making null/no question in one of cells [row, col] ?

Reply:
I am afraid I didn't get your point. Could you please give an example?

Reply:
Valid scenario:
```
[
  [Q, null, Q],
  [null, Q, Q],
  [Q, Q, null]
]
```

Invalid scenario:
```
[
   [Q, Q, Q],
   [Q],
   [Q, Q]
]
```